### PR TITLE
feat: Save shortcut + Vsync

### PR DESCRIPTION
**Pull request by diegusmejia117**
Created feature CTRL S : You can now save with the combination of ctrl + s, it works as standard first save as Save As, subsequent saves will work as should.

Created feature VSYNC toogle: this toggle activates/deactivates the VSYNC functionality.

### CODE REVIEW

File: Source/DataModels/Windows/EditorWindows/WindowAbout.cpp [hunk 1]
```diff
@@ -11,7 +11,7 @@ WindowAbout::~WindowAbout()
 void WindowAbout::DrawWindowContents()
 {
 	ImGui::TextUnformatted(TITLE);
-	ImGui::TextWrapped("Description: nullptr");
+	ImGui::TextWrapped("Description: Axolotl Engine is a game development tool developed by a group of students at the UPC SCHOOL");
```
Comment (markmeliaa) [inline Source/DataModels/Windows/EditorWindows/WindowAbout.cpp:14]:
```suggestion
	ImGui::TextWrapped("Description: Axolotl Engine is a game building tool developed by a group of students at the UPC School");
```

**Review by markmeliaa — CHANGES_REQUESTED** [review]
Nice work, this is a good feature, but just a couple simple changes:

File: Source/DataModels/Windows/EditorWindows/WindowAbout.cpp (hunk 1 above)
Comment (gerard-queralt) [reply Source/DataModels/Windows/EditorWindows/WindowAbout.cpp:14]: If you don't mind, I think it sounds better the other way around. "... game development tool built by..."

**Comment by gerard-queralt** [comment]
This was branched from [create-new-scene-v2](https://github.com/Horizons-Games/Axolotl-Engine/tree/create-new-scene-v2), if I'm not mistaken, so we at least need to merge #219 before merging this.

File: Source/DataModels/Windows/EditorWindows/WindowAbout.cpp (hunk 1 above)
Comment (MarcGonzalezMoratona) [reply Source/DataModels/Windows/EditorWindows/WindowAbout.cpp]: resolved because I have already changed it in [#219]

**Review by gerard-queralt — CHANGES_REQUESTED** [review]

**Review by gerard-queralt — DISMISSED** [review]

**Review by markmeliaa — APPROVED** [review]
All good, thanks!

**Review by gerard-queralt — DISMISSED** [review]

**Comment by markmeliaa** [comment]
Unfortunately, there seems to be a couple conflicts that need to be fixed before merging this.

**Review by AuxPort66 — DISMISSED** [review]

**Review by markmeliaa — APPROVED** [review]

File: Source/DataModels/Windows/WindowMainMenu.cpp [hunk 2]
```diff
@@ -143,3 +143,12 @@ void WindowMainMenu::DrawHelpMenu()
 	about->Draw(showAbout);
 }
 
+void WindowMainMenu::ShortcutSave()
+{
+	std::string filePathName = App->scene->GetLoadedScene()->GetRoot()->GetName();
+	if (filePathName != "New Scene") App->scene->SaveSceneToJson(filePathName + SCENE_EXTENSION);
```
Comment (PaablooCH) [inline Source/DataModels/Windows/WindowMainMenu.cpp:149]: if and else don't follow the programming rules, they need the {}

File: Source/Modules/ModuleInput.cpp [hunk 3]
```diff
@@ -188,6 +200,10 @@ update_status ModuleInput::Update()
 #endif // ENGINE
     }
 
+#ifdef ENGINE
+    if (keysState[SDL_SCANCODE_LCTRL] == KeyState::REPEAT && keysState[SDL_SCANCODE_S] == KeyState::DOWN)
+        App->editor->GetMainMenu()->ShortcutSave();
```
Comment (PaablooCH) [inline Source/Modules/ModuleInput.cpp:205]: doesn't follow the programming rules, should be between {}

File: Source/Modules/ModuleEditor.h [hunk 4]
```diff
@@ -55,6 +56,10 @@ inline const WindowScene* ModuleEditor::GetScene() const
 	return scene;
 }
 
+inline WindowMainMenu* ModuleEditor::GetMainMenu() const
+{
+	return mainMenu.get();
+}
```
Comment (PaablooCH) [inline Source/Modules/ModuleEditor.h:62]: add a space between functions

File: Source/Modules/ModuleInput.cpp [hunk 5]
```diff
@@ -97,6 +98,17 @@ update_status ModuleInput::Update()
 
     const Uint8* keyboard = SDL_GetKeyboardState(NULL);
 
+    
+
+    
+
```
Comment (PaablooCH) [inline Source/Modules/ModuleInput.cpp:104]: too much spaces between lines of code

File: Source/DataModels/Windows/WindowMainMenu.cpp [hunk 6]
```diff
@@ -143,3 +143,12 @@ void WindowMainMenu::DrawHelpMenu()
...
+	else isSaving = true;
+
+
```
Comment (PaablooCH) [inline Source/DataModels/Windows/WindowMainMenu.cpp:152]: you can delete the spaces between code and }

**Review by PaablooCH — CHANGES_REQUESTED** [review]
Before merging a notice some changes with the style

**Review by PaablooCH — APPROVED** [review]

**Review by markmeliaa — APPROVED** [review]

**Pull request merged**